Secure-Software-Development-Policy/README.md

```diff
@@ -0,0 +1,157 @@
+# Zowe Secure Software Development Policy Framework
+
+This document describes a framework used to model a meta definition of the Zowe Secure Development Policy/ies.
+
+The policies described bellow are titled "Meta-Policy", because they serve as a container to collect all the requirements for a given area of governance (eg. Organization, SDLC, etc).
+Such meta policy can/should be split into smaller interrelated policies targeting specific security topics, which need to be governed/published separately (e.g. Security issues disclosure etc.).
+Such target, publishable policies need to be distilled from a Meta Policy. while keeping the relation to the original statements for easier maintenance and assessment of policy fulfilment.
+
+```
+### Goals:
+- Keep it simple and extensible as much as possible
+- Must be complete, i.e contributors could refer it and not need to define their own (but could extend ot if they need)
+- Maintain reference to external requirements for approval of compliance and conformity
+```
+
+![Secure Software Development Policy Framework]("./img/SecurityPolicyFramework.png")
+
+###Common definitions:
+> 1. Contributors' types and policy applicability\
+> There are basically three types of Zowe contributors.
+>- Zowe as an entity represented by ZAC and TSC 
+>- Zowe squads owning component/s
+>- Individual contributors (persons or organizations) contributing to one or multiple Zowe components/projects\
+> 
+ > They not only differ by size and ownership of the components, but consequently have different responsibilities
+ > and should be required to maintain different level of compliance with the policies.
+ > On the top of the responsibility chain is always Zowe represented by the TSC and Security Workgroup.
+ > The question then is how to structure the policy statements so to assign them to corresponding entity (individual, squad, Zowe governance body)\
+> 
+> 2 . Policy statements\
+>Some requirements are strictly mandatory, while some others might not be enforceable in all cases and should be perceived as strong recommendations.
+>
+>* Note: Should we state next to each requirement if it is mandatory or not and under what conditions?
+>* Note: Where is the right place to state what evidence is required to prove fulfilment of mandatory requirements?
```

Comment:
Each item that needs proof should have an explanation of what proof and how to provide it directly next to the text of the requirement. If its too much to explain, just put a link to another page.